src/main/java/com/exasol/adapter/dynamodb/documentnode/dynamodb/DynamodbBinary.java

```diff
@@ -8,15 +8,16 @@
  * This class represents a DynamoDB binary value.
  */
 public class DynamodbBinary implements DocumentValue<DynamodbNodeVisitor> {
-    private final ByteBuffer value;
+    private static final long serialVersionUID = -1085912088779479403L;
+    private final byte[] value;
 
     /**
      * Creates an instance of {@link DynamodbBinary}.
      *
      * @param value value to hold
      */
     public DynamodbBinary(final ByteBuffer value) {
-        this.value = value;
+        this.value = value.array();
```

Comment:
Was there any particular reason to convert this to Array?

Reply:
`ByteBuffer` is not serializable.

Reply:
Thanks makes sense.